src/chats/components/conversation/messageBubbles/BubbleFooter.tsx

```diff
@@ -80,14 +80,14 @@ const BubbleFooter: FC<BubbleFooterProps> = ({
 			crossAlignment="flex-end"
 			padding={{ top: 'small' }}
 		>
-			<Container width="fit" padding={{ right: 'medium' }}>
+			<Row takeAvailableSpace mainAlignment="flex-start" padding={{ right: 'medium' }}>
 				{messageExtension && messageSize && (
 					<Text color="secondary" size="small">
```

Comment:
It's better to use TextWithTooltip DS component instead of Text, so when text is ellipsed a tooltip appears if the user hovers over the text